.github/workflows/translate.yml

```
@@ -0,0 +1,72 @@
+name: Update Translations
+on:
+  push:
+    branches:
+      main
```

I think this needs a dash?
```suggestion
      - main
```